src/Components/InputBook.js

```diff
@@ -1,20 +1,79 @@
-import React, { Fragment } from 'react';
-
-const InputBook = () => (
-  <Fragment>
-    <form >
-      <label htmlFor="Add-new-book">ADD A NEW BOOK</label>
-      <br />
-      <input type="text" placeholder="Book Title" />
-      <select placeholder="Category">
-        <option>Category</option>
-        <option>Author1</option>
-        <option>Author2</option>
-        <option>Author3</option>
-      </select>
-      <button type="submit">ADD BOOK</button>
-    </form>
-  </Fragment>
-);
+import React, { Fragment, useState } from "react";
+import { useDispatch } from "react-redux";
+import { nanoid } from "@reduxjs/toolkit";
+import { AddBook } from "../redux/books/bookSlice";
+import "../Styles/input.css";
+
+const InputBook = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState("");
+  const [author, setAuthor] = useState("");
+  const [categories, setCategory] = useState([
+    "Category",
+    "Science Fiction",
+    "Action",
+    "Economy",
+  ]);
+
+  const onTitleChange = (e) => {
+    setTitle(e.target.value);
+  };
+
+  const onAuthorChange = (e) => {
+    setAuthor(e.target.value);
+  };
+
+  // const onCategoryChange = (e) => {
+  //   setCategory(e.target.value);
+  // };
```

Comment:
- [ ] Please kindly uncomment this code or remove it completely, as it does not make the codebase look neat